java/serve/src/main/java/io/ray/serve/BackendConfig.java

```diff
@@ -0,0 +1,90 @@
+package io.ray.serve;
+
+import com.google.common.base.Preconditions;
+import java.io.Serializable;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+/**
```

Comment:
Let's make sure to convert this into protobuf in the next PR?

Reply:
Yes. I'll design some plans about next PR this week.